src/Sylius/Bundle/CurrencyBundle/Resources/translations/validators.fr.yml

```
@@ -4,20 +4,20 @@
 sylius:
     currency:
         code:
-            exact_length: Le code de la devise doit comporter exactement 3 caractères.
-            not_blank: Veuillez choisir le code de la devise.
-            regex: Le code de la devise peut uniquement être constitué de lettres, chiffres, tirets et de tirets bas.
-            unique: Le code de la devise doit être unique.
+            exact_length: Le code de la monnaie doit avoir exactement 3 caractères.
+            not_blank: Veuillez sélectionner le code de la monnaie.
+            regex: Le code de la monnaie ne peut contenir que des lettres, des chiffres, des tirets et des tirets bas.
+            unique: Le code de la monnaie doit être unique.
```

```suggestion
            exact_length: Le code de la devise doit comporter exactement 3 caractères.
            not_blank: Veuillez choisir le code de la devise.
            regex: Le code de la devise peut uniquement être constitué de lettres, chiffres, tirets et de tirets bas.
            unique: Le code de la devise doit être unique.
```

Please revert this change as the original translations were more accurate

---

Also I don't know what others think but potentially 'tirets bas' might be safer to not translated and kept as "underscores" or maybe something like : 
- tirets bas (_)
- tirets bas (_underscores_)